sdk/formrecognizer/ai-form-recognizer/samples/typescript/src/shims.d.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint  @typescript-eslint/no-empty-interface: 0 */
+
+interface FileReader {
```

Comment:
We can do something similar to what we do in core-http here so that customers don't have to shim FileReader or add dom.d.ts.